<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateClass</class>
 <widget class="QDialog" name="updateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>641</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>641</width>
    <height>448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>年份与周数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>601</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>年份:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_year">
       <item>
        <property name="text">
         <string>未选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2019</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2020</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2021</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2022</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2023</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2024</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2025</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2026</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2027</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2028</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2029</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>周：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cb_week">
       <item>
        <property name="text">
         <string>未选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>当前周次未排课学生</string>
   </property>
   <widget class="QTableView" name="tv_show_mem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>选择排课信息</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>选择教练：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>日期课时</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_coach">
       <item>
        <property name="text">
         <string>未选择</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_day"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>操作菜单</string>
   </property>
   <widget class="QPushButton" name="btn_flush">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新未排课名单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确认排课信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>当前选中</string>
   </property>
   <widget class="QLineEdit" name="le_choose_name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_choose_phone">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
